spec/support/rails_app/db/schema.rb

```diff
@@ -22,6 +22,9 @@
     t.integer "wizard_id", null: false
     t.string "wood_type"
     t.integer "core_type"
+    t.float "flexibility", null: false
```

Comment:
XD these would make my teammates really happy. 
cc @liza8bit @fresh-prince-of-prosemirror

Reply:
Will do! Also I forgot to add `T::Boolean` as a possible JSON value type. Will add that too.

Reply:
Alright, just added the migration and support for `T::Boolean` in json columns.